discovery API.ipynb

```diff
@@ -0,0 +1,1171 @@
+{
```

Comment:
Since Github does not deal well with changes in notebook files, I think it's better to commit the notebooks without the outputs. So let's remove them, please.

Reply:
OK, we added it here only for sharing and feedback gathering but I guess there's no need of that anymore?

Reply:
Yep, that's useful. Let's remove them then before merging.